prov/efa/docs/efa_rdm_protocol_v4.md

```diff
@@ -1281,6 +1354,44 @@ that is not an feature because the peer does not do anything different. Rather,
 endpoint has for the sender. Therefore, it was re-interpreted as an extra request named "constant header length".
 This re-interpretation does not change the implementation, thus does not cause backward incompatibility.
 
+### 4.4 have connection ID in packet header (connid header)
+
+The "have connection ID in packet header" extra request was introduced with libfabric 1.14.0 release, and was
+assigned the ID 3.
+
+This extra feature is designed to solve the "QP collision" problem, which is commonly experienced in
+client-server type of application.
+
+The "QP collision" problem arise from fact that the EFA device uses the gid + qpn as the unique
+identifier of a peer. Recall that raw address of EFA endpoint is consisted of 3 parts:
+gid + qpn + connid. EFA device only recognizes gid and qpn. The connid was generated by the endpoint
```

Comment:
I'd expand the acronyms here for clarity

Reply:
fixed in new revision.